package.json

```
@@ -18,8 +20,8 @@
     "build:firefox": "pnpm clean:bundle && turbo ready && cross-env __FIREFOX__=true turbo build",
     "zip": "pnpm build && pnpm -F zipper zip",
     "zip:firefox": "pnpm build:firefox && cross-env __FIREFOX__=true pnpm -F zipper zip",
-    "dev": "turbo ready && cross-env __DEV__=true turbo watch dev --concurrency 20",
-    "dev:firefox": "turbo ready && cross-env __DEV__=true __FIREFOX__=true turbo watch dev --concurrency 20",
+    "dev": "pnpm clean:bundle && turbo ready && pnpm -F i18n ready:dev && cross-env __DEV__=true turbo watch dev --concurrency 20",
+    "dev:firefox": "pnpm clean:bundle && turbo ready && pnpm -F i18n ready:dev && cross-env __DEV__=true __FIREFOX__=true turbo watch dev --concurrency 20",
```

Does `pnpm clean:bundle` need to be run for every development step? It seems like an unnecessary side effect.
```suggestion
    "dev": "turbo ready && pnpm -F i18n ready:dev && cross-env __DEV__=true turbo watch dev --concurrency 20",
    "dev:firefox": "turbo ready && pnpm -F i18n ready:dev && cross-env __DEV__=true __FIREFOX__=true turbo watch dev --concurrency 20",
```

---

let's try to shuffle between running the `build` script and `dev` script

---

I think it's good to ensure everything is removed before fresh `dev` start

---

Do you think running clean:bundle can remove some sort of side effect? This seems unnecessary to me.

---

If you want i can remove it.

It's useful, when you are developing CORE, like we're doing here, if you want to add new file, but remove it later of sth like that, it'll still exists in `build` and this enforce to remove all bundles to have clear next `dev` run.

But for the most of the users it couldn't be that helpful

---

let's remove it 😄